query-tuning/collector-workflow.mdx

```diff
@@ -0,0 +1,128 @@
+---
+title: 'Setting up the Collector Workflow'
+backlink_href: /docs/query-tuning
+backlink_title: 'Query Tuning'
+---
+
+import CopyCodeBlock from '../components/CopyCodeBlock'
+import DocsBetaFeature from '../components/DocsBetaFeature'
+import { explainAnalyzeHelper } from '../components/helperSQL'
+
+<DocsBetaFeature feature="Query Tuning" />
+
+The Collector Workflow in Query Tuning workbooks optionally allows users with the [Tune Queries permission]()
```

Comment:
I think you can fill `/docs/accounts/permissions#tune-queries` already here as it'll be the link